File: model/playlist.go
package model

import (
	"github.com/jinzhu/gorm"
)

type Playlist struct {
	gorm.Model
	PlaylistID    int64  `gorm: "column: playlist_id; type: bigint; primary_key; unique; not null"`
	PlaylistTitle string `gorm: "column: playlist_title; type: nvarchar(50); unique; not null"`
	CreatedByID   int64  `gorm: "column: created_by_id; type: bigint; not null"`
}

File: admin/backend/src/db/model/playlist.ts
import { Model, snakeCaseMappers } from 'objection';

export class Playlist extends Model {
    playlistId: string;
    playlistTitle: string;
    createdById: string;

    static get tableName() {
        return 'playlists';
    }

    static get idColumn() {
        return 'playlist_id';
    }

    static get columnNameMappers() {
        return snakeCaseMappers();
    }
}

export default Playlist;